charts/civil-service/values.aat.template.yaml

```diff
@@ -61,7 +61,7 @@ java:
     FEIGN_CLIENT_CONFIG_REMOTEREPOSITORYSERVICE_URL: http://${SERVICE_NAME}-camunda/engine-rest/
     SERVICE_REQUEST_UPDATE: http://${SERVICE_FQDN}/service-request-update
     TESTING_SUPPORT_ENABLED: true
-    POLLING_EVENT_EMITTER_ENABLED: false
+    polling.event.emitter.enabled: false
```

Comment:
Is there a "less invasive" way to do this? Like by commenting the line explaining why it's been commented instead of being taken out entirely?

Reply:
it is not taken out @filippo-possenti-version1 - the property name was not used correctly so that is why always defaulting to true, I have changed just the name

Reply:
Got it. All good then.